ops/docker-compose.yml

```diff
@@ -11,26 +11,26 @@ services:
       dockerfile: ./ops/docker/Dockerfile.monorepo
 
   # this is a helper service used because there's no official hardhat image
-  l1_chain:
+  l1geth:
```

Comment:
Can we keep it as `l1_chain`? Hardhat is not a geth instance.

Reply:
Maybe we should just call it `hardhat_l1_network` to be explicit

Reply:
Yeah, I'll revert. nbd.